charts/seqr/templates/deployment.yaml

```diff
@@ -36,6 +36,7 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 6 }}
       {{- end }}
+      serviceAccountName: {{ include "seqr.fullname" . }}
```

Comment:
i feel like this needs to be gated with logic -- if serviceAccount.create is false, and we try to reference one that doesn't exist, i believe k8s can't create the pod/deployment.

Reply:
hrm interesting.  I agree with you, but the helm chart boilerplate didn't gate this line in the same way it did with the service account manifest itself.  I think gating is better though.

Reply:
ah, the helm chart boilerplate looks like it has a special _helper template for the service account name, which resolves to "default" when there's no service account being created. Which, TIL there's always a service account called default:

```
➜ kubectl get sa
NAME       SECRETS   AGE
default    0         26h
```